.github/workflows/reviewdog.yml

```diff
@@ -25,4 +36,4 @@ jobs:
           reporter: github-pr-check
           path: "."
           pattern: "*.sh*"
-          exclude: "./.git/*"
+          exclude: "./.git/*"
```

Comment:
@prologic, how could a vim user like you have done this? No newline at the end? :-)

Reply:
Oh that's easy :)

```
:help eol
```

🤣